aldryn_newsblog/search_indexes.py

```diff
@@ -30,11 +33,17 @@ def index_queryset(self, using=None):
         return qs
 
     def get_index_queryset(self, language):
-        return self.get_model().objects.published().active_translations(
-            language_code=language).filter(app_config__search_indexed=True)
+        return (
+            self.get_model().objects
+                            .published()
+                            .language(language)
+                            .active_translations(language_code=language)
+                            .filter(app_config__search_indexed=True)
+        )
```

Comment:
Parler has method to return only translated objects: http://django-parler.readthedocs.org/en/latest/api/parler.managers.html#parler.managers.TranslatableQuerySet.translated
I wonder if it could be used here?